### PR TITLE
Release action

### PR DESCRIPTION
**Pull request by FNTwin**
Release action for gh on pypi

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+          cache: pip
+
+      #- name: Install requirements
+      #  run: |
+      #    pip install -U pip
+      #    pip install .[dev]
```
Comment (hatemhelal) [inline .github/workflows/release.yaml:39]: delete these comments?
```suggestion
```

**Comment by hatemhelal** [comment]
I'm far out of my own experience here, it might be nice to run one or two "smoke" tests on the built wheel to avoid publishing a broken package, or is that not really a problem in practice?

**Comment by FNTwin** [comment]
Not a problem, merge this commit [848c26d](https://github.com/valence-labs/mess/pull/5/commits/848c26d2fe35cf2c15d89c27332a5f6971456473). I can run the release and test it manually. Then we can enable the release and publish it. (for this kind of things aka setting up , the merge blocking is really a chore)

**Review by hatemhelal — APPROVED** [review]
Trust!

File: .github/workflows/release.yaml (hunk 1 above)
Comment (FNTwin) [reply .github/workflows/release.yaml:39]: I'm doing the install after in a different way. Going to rebase, push and merge

**Pull request merged**